plaidml/bridge/openvino/src/plaidml_plugin/plaidml_executable_network.cpp

```diff
@@ -33,7 +33,7 @@ InferRequestInternal::Ptr PlaidMLExecutableNetwork::CreateInferRequestImpl(Input
     IVLOG(2, "output: " << kvp.first);
     outputs.push_back(tensorIOMap_.at(kvp.first));
   }
-  auto program = edsl::ProgramBuilder("ie", outputs).target("llvm_cpu").compile();
+  auto program = edsl::ProgramBuilder("ie", outputs).compile();
```

Comment:
@tzerrell Note, you'll have to ensure `PLAIDML_TARGET` is set properly, one way to ensure this is to run `//plaidml:setup`.